apps/src/codebridge/FileBrowser/FileBrowserRow/hooks/useFileRowOptions.ts

```diff
@@ -90,6 +94,25 @@ export const useFileRowOptions = (
         labelText: codebridgeI18n.renameFile(),
         clickHandler: () => openRenameFilePrompt({fileId: file.id}),
       },
+      {
+        condition: enableUserAddedContext(appName, file),
```

Comment:
I wonder if we should display this option if the user is not in the AI tutor tab.

Reply:
Maybe a question for @dmcavoy